docs/integrations/sources/datadog-migrations.md

```diff
@@ -0,0 +1,6 @@
+# Datadog Migration Guide
+
+## Upgrading to 1.0.0
+
+Spec and schema are inline now, and default start and end date is setup for incremental sync.
+Currently the manifest is compatible with yaml editor of builder, After issue `https://github.com/airbytehq/airbyte/issues/44374` is resolved, the manifest would be fully compatible with the builder UI.
```

Comment:
```suggestion
Starting 1.0, Datadog source will apply start and end dates to sync data incrementally. When upgrading, take a minute to set start and end date config options.
```

Reply:
The reasoning here is that the end user doesn't care about inline schemas or Builder that much — in this doc they need to understand how to setup their connector